hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/http/ContentTypes.java

```diff
@@ -0,0 +1,33 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.hadoop.http;
+
+/**
+ * Content Types for HTTP.
+ *
+ * @since 3.5.0
```

Comment:
cut so it's easier to backport

Reply:
sure - the change will probably get a merge conflict when #6554 is merged so I might wait until that is merged

The reason for the likely merge conflict is that in a few classes the imports for the 2 java.ws classes are beside each other.

Reply:
done